Sharpmake/Project.Configuration.cs

```diff
@@ -2166,6 +2166,17 @@ public enum ProjectFilter
                 /// </summary>
                 public string Executable = "";
                 /// <summary>
+                /// This passes the Executable string through a resolver
+                /// </summary>
+                /// <remarks>
+                /// Set this to false to output Executable string without any changes
+                /// </remarks>
+                public bool ResolveExecutable = true;
```

Comment:
We should probably only have a single boolean named something like bool UseExecutableFromSystemPath and test this variable for both cases where you were using your two new bools.

Reply:
Perfect, I will get on this, I have the weekend to get this done.

Reply:
I just pushed this change.